NEWS.md

```diff
@@ -1,4 +1,20 @@
-## 10.0.0 - Unreleased
+## 10.1.0 - Unreleased
+
+## 10.0.0 - Released
+The main focus of this release was to introduce required fields in schemas and adding new CRUD API
```

Comment:
not a big deal but it would be nice to be explicit about which API was added...

... " and introducing the titles API"